xml/http_servers.xml

```diff
@@ -1889,22 +1889,73 @@
+      <param pos="0" name="service.component.product" value="Reverse Channel Proxy"/>
+      <param pos="1" name="service.component.version"/>
    </fingerprint>
 
-   <fingerprint pattern = "^com.hp.openview.bbc.LLBServer (\d\.\d.\d\.\d)$">
-      <description>HP Openview LLBServer</description>
+   <fingerprint pattern = "^(?:BBC \d+\.\d+\.\d+\.?\d*; )?com.hp.openview.bbc.LLBServer (\d+\.\d+\.\d+\.?\d*)$">
+      <description>HP Openview LLBServer (Local Location Broker)</description>
       <example>com.hp.openview.bbc.LLBServer 2.6.8.1</example>
+		<example>BBC 2.6.0.7; com.hp.openview.bbc.LLBServer 2.6.0.7</example>
       <param pos="0" name="service.vendor" value="HP"/>
       <param pos="0" name="service.family" value="OpenView"/>
-      <param pos="0" name="service.product" value="OpenView"/>
-      <param pos="1" name="service.version"/>
+      <param pos="0" name="service.component.vendor" value="HP"/>
+      <param pos="0" name="service.component.family" value="OpenView"/>
+      <param pos="0" name="service.component.product" value="LLBServer"/>
+      <param pos="1" name="service.component.version"/>
+   </fingerprint>
+
+   <fingerprint pattern="^BBC \d+\.\d+\.\d+; ovbbccb (\d+\.\d+\.\d+)$">
+      <example>BBC 06.00.083; ovbbccb 06.00.083</example>
+      <example>BBC 06.20.050; ovbbccb 06.20.050</example>
+      <example>BBC 11.10.035; ovbbccb 11.10.035</example>
+      <description>OpenView Communication Broker (ovbbccb)</description>
+      <param pos="0" name="service.vendor" value="HP"/>
+      <param pos="0" name="service.family" value="OpenView"/>
+      <param pos="0" name="service.component.vendor" value="HP"/>
+      <param pos="0" name="service.component.family" value="OpenView"/>
+      <param pos="0" name="service.component.product" value="Communication Broker"/>
+      <param pos="1" name="service.component.version"/>
+   </fingerprint>
+
+	<fingerprint pattern="^BBC \d+\.\d+\.\d+; ovbbccb unknown version$">
+      <example>BBC 11.13.007; ovbbccb unknown version</example>
+      <description>OpenView Communication Broker (ovbbccb) with no version</description>
+      <param pos="0" name="service.vendor" value="HP"/>
+      <param pos="0" name="service.family" value="OpenView"/>
+      <param pos="0" name="service.component.vendor" value="HP"/>
+      <param pos="0" name="service.component.family" value="OpenView"/>
+      <param pos="0" name="service.component.product" value="Communication Broker"/>
+   </fingerprint>
+
+   <fingerprint pattern="^UOS$">
+      <example>UOS</example>
+      <description>HTTP Server that appears unique to Managment Console on HP TippingPoint IPS Devices</description>
+      <param pos="0" name="service.vendor" value="HP"/>
+      <param pos="0" name="service.product" value="HTTP"/>
+      <param pos="0" name="service.family" value="TippingPoint"/>
+      <param pos="0" name="os.vendor" value="HP"/>
+      <param pos="0" name="os.family" value="TippingPoint"/>
```

Comment:
I am assuming you didn't put a product here because you can't know any more than "yes, this is a TippingPoint device of some sort".  In that case, maybe adding an {{os.device}} of {{IPS}} might be good.  I'll get that and other cleanups to you in a PR in a moment.